src/index.mako.html

```diff
@@ -51,23 +51,23 @@
     <meta property="og:description" content="geo.admin.ch ist die Geoinformationsplattform der Schweizerischen Eidgenossenschaft. // geo.admin.ch est la plateforme de géoinformation de la Confédération suisse."/>
     <meta property="og:type" content="WebApplication"/>
     <meta property="og:url" content="//map.geo.admin.ch"/>
-    <meta property="og:image" content="${s3basepath}${versionslashed}img/logo_geoportal.png"/>
+    <meta property="og:image" content="//public.geo.admin.ch/img/logo_geoportal.png"/>
```

Comment:
I would put https directly in the URL - not sure if cards handle it correct. As well for the og:url part. It's like this currently on prod. I seem to remember troubles if its not full url.